app/kumactl/cmd/get/get_mesh.go

```diff
@@ -0,0 +1,65 @@
+package get
+
+import (
+	"context"
+	"io"
+
+	"github.com/pkg/errors"
+
+	"github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Kong/kuma/app/kumactl/pkg/output"
+	"github.com/Kong/kuma/app/kumactl/pkg/output/printers"
+	rest_types "github.com/Kong/kuma/pkg/core/resources/model/rest"
+	"github.com/Kong/kuma/pkg/core/resources/store"
+)
+
+func newGetMeshCmd(pctx *getContext) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "mesh NAME",
+		Short: "Show Single Mesh Resource",
+		Long:  `Show Single Mesh Resource.`,
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			rs, err := pctx.CurrentResourceStore()
+			if err != nil {
+				return err
+			}
+			name := args[0]
+			currentMesh := name
+			mesh := mesh.MeshResource{}
+			if err := rs.Get(context.Background(), &mesh, store.GetByKey(name, currentMesh)); err != nil {
+				if store.IsResourceNotFound(err) {
+					return errors.Errorf("No resources found in %s mesh", currentMesh)
+				}
+				return errors.Wrapf(err, "failed to get mesh %s", currentMesh)
+			}
+			switch format := output.Format(pctx.args.outputFormat); format {
+			case output.TableFormat:
+				return printMesh(&mesh, cmd.OutOrStdout())
+			default:
+				printer, err := printers.NewGenericPrinter(format)
+				if err != nil {
+					return err
+				}
+				return printer.Print(rest_types.From.Resource(&mesh), cmd.OutOrStdout())
+			}
+		},
+	}
+	return cmd
+}
+
+func printMesh(mesh *mesh.MeshResource, out io.Writer) error {
+	data := printers.Table{
+		Headers: []string{"MESH", "NAME"},
+		NextRow: func() []string {
```

Comment:
I guess `NextRow` should return `nil`, after the last element. Please take a look here: https://github.com/Kong/kuma/blob/master/app/kumactl/cmd/get/get_dataplanes.go#L48

Reply:
Thanks @lobkovilya fixed it